contrib/msggen/msggen/schema.json

```diff
@@ -23999,7 +23999,25 @@
                 "max_total_htlc_in_msat": {
                   "type": "msat",
                   "description": [
-                    "Max amount accept in a single payment."
+                    "Max amount accept in a single payment. This field is deprecated, use instead our_max_htlc_value_in_flight_msat"
```

Comment:
nit: this commit should be squashed into the commit where these fields are originally introduced/added to the RPC (the first one in the series, in this case)

Reply:
It's complicated. These are autogenerated, if you ask me I wouldn't have these files into the source code cause no human wrote them. I add them as a separate commit to make it easier to rebase on the master branch in case someone else has also modified them.

My usual rebase when changing schemas is
- soft reset the last commit (with the autogenerated files, very annoying),
- rebase
- then generate those files again and commit